### PR TITLE
sed: add 11q example

### PR DESCRIPTION
**Pull request by Ray6464**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples. (it was over 8 already + filled with a ton of examples that are not tldr)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/sed.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 `sed '/{{line_pattern}}/d' {{filename}}`
 
+- Print the first 11 lines of a file
```
Comment (navarroaxel) [inline pages/common/sed.md:25]: This example is covered with the last example.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @Ray6464 thanks for your submission, but the limit is 8 examples per command.

**Review by sbrl — COMMENTED** [review]
Hey there, @Ray6464! Unfortunately we have a limit of 8 example per page. Is there a less useful example we could remove instead?

Also, don't forget the `{{token_syntax}}` to indicate what the user can change :slightly_smiling_face:

**Comment by Ray6464** [comment]
> Hey there, @Ray6464! Unfortunately we have a limit of 8 example per page. Is there a less useful example we could remove instead?
> 
> Also, don't forget the `{{token_syntax}}` to indicate what the user can change slightly_smiling_face

I am hoping for you guys to figure it out; in my opinion, examples (1) and (2) are very redundant so keep (1) and remove (2) to make room for my PR.
    1) Replace all occurrences of an extended regular expression in a file, and print the result:
    2) Replace all occurrences of a string in a file, overwriting the file (i.e. in-place):

**Comment by sbrl** [comment]
Hrm, I'm inclined to disagree. One oft he goals of tldr-pages is to help the beginner, so I would argue that the first 2 examples are important to help new users to understand thee syntax of the command.

If you want to print the first N lines of a file, wouldn't  `head` be the more appropriate command to use here?

**Comment by Ray6464** [comment]
> Hrm, I'm inclined to disagree. One oft he goals of tldr-pages is to help the beginner, so I would argue that the first 2 examples are important to help new users to understand thee syntax of the command.
> 
> If you want to print the first N lines of a file, wouldn't `head` be the more appropriate command to use here?

I agree with you on the 1st point, however `head` is not a better tool for this for the following reasons:
1) Head is part of the GNUcoreutils package, a malware that has infected almost every single server in existence.
2) Sed is meant to stand on its own.

I'll say either drop the example or make a file length exception of programs like `sed`.

**Comment by sbrl** [comment]
That's a bit extreme to say that the GNU coreutils is malware. Despite that, I do see your point about your `sed` example. However, it's always awkward with commands such as `sed`, as there is so much functionality to fit into such a small page. The purpose of tldr-pages is to distil this down into 8 examples or less, and demonstrate only the most useful functionality. To this end, I'm afraid we can't lift the 8 example limit.

**Comment by Ray6464** [comment]
> That's a bit extreme to say that the GNU coreutils is malware. Despite that, I do see your point about your `sed` example. However, it's always awkward with commands such as `sed`, as there is so much functionality to fit into such a small page. The purpose of tldr-pages is to distil this down into 8 examples or less, and demonstrate only the most useful functionality. To this end, I'm afraid we can't lift the 8 example limit.

How about this, let's argue that 'Print only the n-th line of a file' example is requndant after examples like 'Print the first 11 lines of a file' and 'Print only text between n-th line till the next empty line', so let's remove the `Print only the n-th line of a file` example with this one.

Also, note that after a few iterations the examples will probably be factored into examples of search, edit, delete, etc. and currently we're giving multiple examples of similar operations. TLDR page of Sed is looking almost similar to Man page like examples.

**Comment by sbrl** [comment]
I think we need another opinion on this.

/cc @agnivade, or maybe @owenvoke perhaps?

**Comment by agnivade** [comment]
Yes, we do have some pages which exceed the limit of 8. But that doesn't create a precedent to add yet more examples to it. I'd argue to actually remove one to bring it to 8. :)

This sed page does indeed have a lot more than we actually need. I think that's primarily because sed is such a huge command that can be used in so many ways. It might appear that we have multiple examples of similar operations, but there are subtle differences between each of them and it's mainly geared towards beginners trying to understand the command. So we carefully start with simple things and go towards more complex ones.

But the primary usage of sed that I have seen is in editing text. So I'd say all the removals should be the print examples. From the 3:

- Print only the n-th line of a file:
- Print only text between n-th line till the next empty line:
- Print first 11 lines:

I'd say 2nd is very advanced and less common than the other 2. So that's an easy one to reject. Among 1 and 3, it's a bit hard, but I'd say 1 would be slightly less common as a use-case than 3.

My suggestion would be to remove 1 and 2, and bring down the total count to 8.

**Comment by Ray6464** [comment]
> My suggestion would be to remove 1 and 2, and bring down the total count to 8.

I'm up for this!

File: pages/common/sed.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/sed.md:25]:
```suggestion
- Print the first 11 lines of a file:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]
LGTM

**Review by owenvoke — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Pull request merged**